bash/code_to_gast/bash_helpers.js

```diff
@@ -0,0 +1,46 @@
+const bash_router = require("./bash_router.js")
+
+
+function script_to_gast(node) {
+    gast = {"type": "root"}
+    gast["body"] =  bash_router.node_to_gast(node["commands"])
+    return gast
+}
+
+function array_to_gast(node) {
+    let node_list = []
+    for(const elem of node) {
+        node_list.push(bash_router.node_to_gast(elem))
+    }
+    return node_list
+}
+
+function word_to_gast(node) {
+    text = node["text"]
+    if (text === "true") {
```

Comment:
Why is text being handled within this function as opposed to by the router? Could you just return code_to_gast(text) and then have a separate function rather than dealing with it within here?

Reply:
This is an attribute of the word node, and word nodes are the base cases, I will add a comment so it is more clear